source/routing/circle.cpp

```diff
@@ -0,0 +1,98 @@
+/* Copyright © 2001-2014, Canal TP and/or its affiliates. All rights reserved.
+
+This file is part of Navitia,
+    the software to build cool stuff with public transport.
+
+Hope you'll enjoy and contribute to this project,
+    powered by Canal TP (www.canaltp.fr).
+Help us simplify mobility and open public transport:
+    a non ending quest to the responsive locomotion way of traveling!
+
+LICENCE: This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+Stay tuned using
+twitter @navitia
+IRC #navitia on freenode
+https://groups.google.com/d/forum/navitia
+www.navitia.io
+*/
+
+#include "type/geographical_coord.h"
+#include "circle.h"
+
+#include <set>
+#include <assert.h>
+#include <vector>
+#include <boost/geometry.hpp>
+#include <boost/geometry/geometries/polygon.hpp>
+#include <cmath>
+
+namespace navitia { namespace routing {
+
+
+type::GeographicalCoord project_in_direction(const type::GeographicalCoord& center,
+                                             const double& direction,
+                                             const double& radius){
+    assert(radius > 0);
+    double alpha=radius / navitia::type::GeographicalCoord::EARTH_RADIUS_IN_METERS;
```

Comment:
you should put spaces around the `=`

and for the method to be a bit less verbose you can do 

`using navitia::type::GeographicalCoord;` at the begining of the method (or in the file since it's a source file, but you seem to use it only in this method, so it's better to put it in the method)